README.md

```diff
@@ -286,7 +286,7 @@ return from a function call which occurred during an actual program execution.
 
 Each event object has the following attributes:
 
-* **id** *Required* unique identifier. Example: 23522.
+* **id** *Required* unique and monotonous identifier. That is that the id of any given event should be greater than the ids of all its predecessors. Example: 23522.
```

Comment:
I think you mean _increasing_ because monotonous is non-decreasing.

Reply:
Lemme be pedantic for a sec. If they are unique then increasing is the same as monotonous. But you are right this can be reworded. BTW I don't mind closing this if we decide to not include implicit constraints in the spec.

Reply:
@lachrist if you're insisting on being pedantic, decreasing ids are also unique and monotonic.

(BTW, I believe 'monotonic' is the word you meant; 'monotonous' is "dull, tedious, and repetitious; lacking in variety and interest." Which might be true of the ids, but I'm not sure we need to spell it out in the spec ;)
English can be tricky.)